runtime/cmake/options.cmake

```diff
@@ -54,6 +54,12 @@ j9vm_shadowed_option(J9VM_GC_BATCH_CLEAR_TLH "Zero any TLH allocated")
 j9vm_shadowed_option(J9VM_GC_COMBINATION_SPEC "Set on specs which implement LIR 16325:  reduce memory footprint by combining multiple sidecars into the same set of libraries.")
 j9vm_shadowed_option(J9VM_GC_CONCURRENT_SWEEP "Enable concurrent sweep in Modron")
 j9vm_shadowed_option(J9VM_GC_DEBUG_ASSERTS "Specialized GC assertions are used instead of standard trace asserts for GC assertions")
+
+# When enabled, a contiguous block of memory is created for each array which data surpasses the size of a region. This contiguous block represents the array as
```

Comment:
Is this comment copied from somewhere? 
There is a missing word in "for each array which data". Could be "for each array **in** which data"